geopandas/plotting.py

```diff
@@ -114,6 +114,15 @@ def plot_series(s, cmap='Set1', ax=None, linewidth=1.0, figsize=None, **color_kw
 
         matplotlib axes instance
     """
+    if 'colormap' in color_kwds:
+        warnings.warn("'colormap' is deprecated, please use 'cmap' instead "
+                      "(for consistency with matplotlib)", FutureWarning)
+        cmap = color_kwds.pop('colormap')
```

Comment:
Well, I also thought about that, and in principle we could easily raise a warning or error in such a case, but I don't know that is really necessary